### PR TITLE
Test updates

### PR DESCRIPTION
**Pull request by kilnerm**
Update test script to check image overrides are reflected.
Add test config for image overrides.
Minor test tweaks for usability.

### CODE REVIEW

File: tests/run_tests.sh [hunk 1]
```diff
@@ -17,21 +19,23 @@ declare -a succesful_tests
 declare -a failed_tests
 test_count=0
 for test_file in $test_dir/*_test.yaml; do
+    test_filename=$(basename $test_file)
+    test_filename="${filename%.*}"
```
Comment (groeges) [inline tests/run_tests.sh]: You are storing values in the same variable, so the fist setting is never used. Is this intentional.

Comment (kilnerm) [reply tests/run_tests.sh]: Yes, as I don't want the filename extension. I could maybe consolidate to one command.

File: tests/run_tests.sh [hunk 2]
```diff
@@ -76,7 +90,37 @@ for test_file in $test_dir/*_test.yaml; do
...
+                    if [[ "$expected_image_registry" != null ]]; then
+                        if [[ "$expected_image_registry" != "$result_reg" ]]; then
+                            echo "Image registry mismatch, expected: $expected_image_registry, actual: $result_reg"
+                            success="false"
+                            break
+                        fi
+                    fi
+                    if [[ "$expected_image_org" != null ]]; then
+                        if [[ "$expected_image_org" -ne "$result_org" ]]; then
+                            echo "Image organisation mismatch, expeted: $expected_image_org, actual: $result_org"
```
Comment (groeges) [inline tests/run_tests.sh]: Spelling mistake. "expected" should be "expected"

File: tests/run_tests.sh (hunk 1 above)
Comment (kilnerm) [reply tests/run_tests.sh]: Wait, I see the bug.

**Review by groeges — APPROVED** [review]
LGTM

**Pull request merged**